.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -33,6 +33,8 @@ If applicable, add screenshots to help explain your problem.
  - Version AmazonChimeSDKMedia: [e.g. 0.4.0]
  - Can you reproduce this in the demo app?
 If you are reporting a crash:
+ - How often is this crash occurring? Is it reproducible?
```

Comment:
`Is it reproducible?` is kind of redundant of `Can you reproduce this in the demo app?`.
`What is occurring in the app when the crash occurs?` is kind of redundant of repro steps.